### PR TITLE
lets try with specified python

### PR DESCRIPTION
**Pull request by jerowe**

### CODE REVIEW

**Comment by jerowe** [comment]
Ping @conda-forge/core - how do I get this to build against python2 ?

**Comment by scopatz** [comment]
It seems like master should already build against python2, not sure why it wouldn't

**Comment by jerowe** [comment]
What can I do to get these built against python2? For now I have them in my own channel, but that is not ideal.

**Comment by jerowe** [comment]
@scopatz, any word. These have been sitting here for some time. What can I do to get these into the conda-forge channel?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -21,14 +21,14 @@ build:
 
 requirements:
   build:
-    - python
+    - python ==python2.7
```
Comment (ocefpaf) [inline recipe/meta.yaml]: This should not be needed since you have a skip statement above to explicitly avoid Python 3.

Comment (jerowe) [inline recipe/meta.yaml]: I was just trying it out as a PR only in this branch. It got me no joy, and is not on the master branch. I will delete this PR if it makes things better though. ;-)

**Comment by ocefpaf** [comment]
> @scopatz, any word. These have been sitting here for some time. What can I do to get these into the conda-forge channel?

I got a version of this recipe working and I will submit a PR soon but `conda search easybuild` cannot find any of the dependencies, which means they are problematic too since I can see feedstocks for them. Let me see what I can do...

**Comment by ocefpaf** [comment]
Closing this as the changes here do not solve the problem. See https://github.com/conda-forge/easybuild-feedstock/pull/2#discussion_r104926283 for more details on the problem (which started with `vsc-base`).

**Pull request closed without merging**